### PR TITLE
test: Remove redundant PERMUTE_ARGS from fail_compilation tests

### PR DESCRIPTION
**Pull request by ibuclaw**
The d_do_test runner doesn't bother with doing any permutations on these tests.

### CODE REVIEW

**Review by MoonlightSentinel — APPROVED** [review]

**Review by wilzbach — APPROVED** [review]

File: test/fail_compilation/fail19890a.d [hunk 1]
```diff
@@ -1,8 +1,7 @@
 /*
-PERMUTE_ARGS:
 TEST_OUTPUT:
 ---
-fail_compilation/fail19890a.d(9): Error: `void[$n$$?:64=LU$]` size 1 * $n$ exceeds $?:windows+32=0x1000000|0x7fffffff$ size limit for static array
+fail_compilation/fail19890a.d(8): Error: `void[18446744073709551615LU]` size 1 * 18446744073709551615 exceeds 0x7fffffff size limit for static array
```
Comment (Geod24) [inline test/fail_compilation/fail19890a.d:4]: Nope

File: test/fail_compilation/fail19890b.d [hunk 2]
```diff
@@ -1,8 +1,7 @@
 /*
-PERMUTE_ARGS:
 TEST_OUTPUT:
 ---
-fail_compilation/fail19890b.d(9): Error: `void[$n$$?:64=LU$]` size 1 * $n$ exceeds $?:windows+32=0x1000000|0x7fffffff$ size limit for static array
+fail_compilation/fail19890b.d(8): Error: `void[18446744073709551614LU]` size 1 * 18446744073709551614 exceeds 0x7fffffff size limit for static array
```
Comment (Geod24) [inline test/fail_compilation/fail19890b.d:4]: Nope

File: test/fail_compilation/fail19913.d [hunk 3]
```diff
@@ -1,8 +1,7 @@
-/* PERMUTE_ARGS:
  * TEST_OUTPUT:
 ---
-fail_compilation/fail19913.d(11): Error: no property `b` for type `int`
-fail_compilation/fail19913.d(11): Error: mixin `fail19913.S.b!()` is not defined
+fail_compilation/fail19913.d(1): Error: declaration expected, not `*`
+fail_compilation/fail19913.d(12): Error: unrecognized declaration
```
Comment (Geod24) [inline test/fail_compilation/fail19913.d:4]: What ?

**Comment by Geod24** [comment]
I guess you used auto update, and it doesn't seem to support regex because it replaced them with the actual values.

File: test/fail_compilation/fail19890a.d (hunk 1 above)
Comment (ibuclaw) [reply test/fail_compilation/fail19890a.d:4]: I've just gotten round to looking at the test results.  AUTO_UPDATE=1 does a rubbish job at replacing regexes.

Comment (wilzbach) [reply test/fail_compilation/fail19890a.d]: CC @MoonlightSentinel what do you think of simply ignoring test outputs with an regex for the auto updating?

Comment (MoonlightSentinel) [reply test/fail_compilation/fail19890a.d]: Thought the same, it's impossible to automate those.

File: test/fail_compilation/imphint.d [hunk 4]
```diff
@@ -1,50 +1,114 @@
-/* PERMUTE_ARGS:
 TEST_OUTPUT:
```
Comment (MoonlightSentinel) [inline test/fail_compilation/imphint.d:1]:
```suggestion
/*
TEST_OUTPUT:
```

**Review by MoonlightSentinel — CHANGES_REQUESTED** [review]

**Review by MoonlightSentinel — APPROVED** [review]

**Pull request merged**